spec/cassettes/ups_json/address_classification/commercial_address.yml

```diff
@@ -0,0 +1,71 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: https://wwwcie.ups.com/api/addressvalidation/v1/2
+    body:
+      encoding: UTF-8
+      string: '{"XAVRequest":{"Request":{"RequestOption":"2"},"AddressKeyFormat":[{"AddressLine":["800
+        2nd Ave"],"ConsigneeName":"Company","CountryCode":"US","PoliticalDivision1":"New
+        York","PoliticalDivision2":"New York","PostcodePrimaryLow":"10017"}]}}'
+    headers:
+      Accept:
+      - application/json
+      User-Agent:
+      - rest-client/2.1.0 (darwin22 arm64) ruby/3.2.2p53
+      Authorization:
+      - Bearer eyJraWQiOiI2NGM0YjYyMC0yZmFhLTQzNTYtYjA0MS1mM2EwZjM2Y2MxZmEiLCJ0eXAiOiJKV1QiLCJhbGciOiJSUzM4NCJ9.eyJzdWIiOiJ1cHNkZXZlbG9wZXJAY2FuZGxlc2NpZW5jZS5jb20iLCJjbGllbnRpZCI6ImU0M1BCRG1QbTZOdTRTNnlFdXlNQnYyTEN3M2J1RVoyQVhFMFRITEpsQmo2TXFzUyIsIm1lcl9pZCI6Ilk5MzE5VyIsImlzcyI6Imh0dHBzOi8vYXBpcy51cHMuY29tIiwidXVpZCI6IjQzNkNERTAwLThEM0UtMUJGRS1BNURCLTkyREM3QUU3MjVCOSIsInNpZCI6IjY0YzRiNjIwLTJmYWEtNDM1Ni1iMDQxLWYzYTBmMzZjYzFmYSIsImF1ZCI6Ijc4Mjk0NWQ0LTk1NTQtNDZkMi04NjYwLTZmZDdlYWNkNDkwMSIsImF0IjoiWTBEb0NhM3poemo5R3N5TWFhY1ExTVNTVWZzbCIsIm5iZiI6MTcwNjgwNjExNywic2NvcGUiOiJMb2NhdG9yV2lkZ2V0IiwiRGlzcGxheU5hbWUiOiJDYW5kbGVTY2llbmNlIFNoaXBwaW5nIEFwcCIsImV4cCI6MTcwNjgyMDUxNywiaWF0IjoxNzA2ODA2MTE3LCJqdGkiOiI1MTRlZTllOC1kNzY2LTQ2MDEtODc5Yi0zNTFlZThhZWNiYmMifQ.Yo0vHPdWGidRcqvht35yQw0X0fBj_CETYLuoS6Vq94FEBe-7kat1rN_e9rC-3oCZgXKvJiSBop8_asaw6P__eCO-3y337WGgtvXSBWZGLuTWD6fVIuXczb-OwiPKEc1u7vffd60DkQo4OR9iBDPR_zi2AnV7K-bBoo3PtAVPpgg5CejFTcZzt_Vq-RXAhfpn89Tt_8Ip1nUAOIgF08LZBcU38F_BgHCP5Hh5Yqkci62_z32Yg8q_LRNueyNx0vWmujUoxJBBmsdlJ67ltU4havu8CsQDAD8vYZTCcxHej9QG5DLHcpkCRmXKq5iDVTXoWzrBuf7FnDiKfCaApWuOQ6Iihh49KAQDPIg9spvunEimGJJP3PCMeMVR9TjMx_wSgxqjVeYKoeVlIHbEoiIkX6FH9m_KMGpkBnFCNKqvODt4CSoTtPW8ELQbl2oaruNdwO_1FH-kVScz81geebPhGkdAC8WZ78T3AwyasV3hIrUdtDzWGzZkPMP4QSg835mxxVeqicjCEHXLfw5YedLnjpcexZlJEkVZL8UdOF3e8w8-JQhCwvOMu_BK11WVPgIi_aRq6HjrOL2PFSLyY6m2bUcTgJxubg0-Wmu2c5jUaCjneS3EHoOvyxjseyyHVF6-wA0Ci8VEVySZY-YXThKzGB_BLof616ztSf52VLwClkM
```

Comment:
Ideally we would be omitting auth tokens from these cassettes.

Reply:
That's weird. I pushed these up with the token and then force pushed the replacement but you got the old one.

I will see if vcr has a good way to filter these dynamic headers.

Reply:
There is a decent way to filter these out in the cassette. I may need to rename the UPS_ACCESS_TOKEN to GENERIC_TOKEN or something like that so it is not misleading.

Reply:
Done